src/Compilers/Core/Portable/Syntax/SyntaxNodeOrTokenList.cs

```diff
@@ -78,10 +78,24 @@ public static SyntaxNodeOrTokenList Create(ReadOnlySpan<SyntaxNodeOrToken> nodes
             if (nodesAndTokens == null)
```

Comment:
Wait, I think we missed this in the last PR, a span shouldn't be compared to null.

Reply:
so this null check existed from before.  and can be hit here if you pass in an array (and the array is null).  comparing a span to null is fine for this purpose.

Reply:
Looked over the behavior of this API, and behavior of the public APIs that called it a bit. I think the behaviors of the APIs as they stand make sense.

However, runtime has [guidance](https://github.com/dotnet/runtime/issues/84265#issuecomment-1650277489) that `nodesAndTokens == default` should be used instead of `nodesAndTokens == null`. Consider making that change in this PR.

Reply:
sure!

Reply:
```suggestion
            if (nodesAndTokens == default)
```